cluster/mesos/docker/docker-compose.yml

```diff
@@ -85,6 +85,7 @@ apiserver:
     --service-node-port-range=30000-32767
     --cloud-provider=mesos
     --cloud-config=/opt/mesos-cloud.conf
+    --allow-privileged
```

Comment:
To answer your specific question: it's not needed for remote exec, only for the PrivilegePod e2e tests.